wv2util/RuntimeList.cs

```diff
@@ -56,6 +56,10 @@ public string Channel
                 {
                     return "Stable";
                 }
+                else if (ExePath.ToLower().Contains("\\edgewebview\\"))
+                {
+                    return "Stable WebView2 Runtime";
```

Comment:
There's a list in the SortUtils of channel names in the order they should get sorted by. Would you mind adding this there too? (I should have had a consolidated list of them somewhere =))